features/admin.actions.v1/components/pre-issue-access-token-action-config-form.tsx

```diff
@@ -113,7 +113,7 @@ const PreIssueAccessTokenActionConfigForm: FunctionComponent<PreIssueAccessToken
 
     // TODO: Temporary flag to show/hide the rule component.
     const showRuleComponent: boolean = isFeatureEnabled(
-        actionsFeatureConfig, "actions.types.preIssueAccessToken.edit.rule");
+        actionsFeatureConfig, ActionsConstants.FEATURE_DICTIONARY.get("PRE_ISSUE_ACCESS_TOKEN_RULE"));
```

Comment:
Can we have something like PRE_PASSWORD_UPDATE_RULE ?
I think we will have to keep enabling rules in a different iterations than the action throughout.

Reply:
I already added another config for the `PRE_UPDATE_PASSWORD_RULE` for pre update password action rule
This config is for the pre issue access token action rule.